src/rgw/rgw_reshard.cc

```diff
@@ -869,11 +869,12 @@ int RGWReshard::list(int logshard_num, string& marker, uint32_t max, std::list<c
   int ret = cls_rgw_reshard_list(store->getRados()->reshard_pool_ctx, logshard_oid, marker, max, entries, is_truncated);
 
   if (ret < 0) {
+    lderr(store->ctx()) << "ERROR: failed to list reshard log entries, oid=" << logshard_oid << " " 
+    << "marker=" << marker << " " << cpp_strerror(ret) << dendl;
```

Comment:
I think you want `cpp_strerror(-ret)`.

Reply:
Never mind.

Reply:
@ivancich can you please change it to approved so we can merge it?

Thanks